test/01LessonOneTest.js

```diff
@@ -2,39 +2,60 @@
 
 var assert = require('assert');
 
+//Homework: read through lesson one. Complete the lesson 1 test. Read the pigLatin.
 
 function returnTrue() {
     // should return true
-
+    return true;
 }
 
 function returnFalse() {
     // should return false
-
+    return false;
 }
 
 function greaterThan(big, small) {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
-
+    if(big > small) {
```

Comment:
No need to have an if, else statement on functions that just test if two values are true or false. I could have used:
`big > small`